tensorflow/lite/micro/tools/ci_build/test_all.sh

```diff
@@ -80,4 +80,7 @@ tensorflow/lite/micro/tools/ci_build/test_arduino.sh
 echo "Running cortex_m_generic tests at `date`"
 tensorflow/lite/micro/tools/ci_build/test_cortex_m_generic.sh
 
+echo "Running cortex_m_corstone_300 tests at `date`"
+tensorflow/lite/micro/tools/ci_build/test_cortex_m_corstone_300.sh
```

Comment:
this seems to be adding ~4 mins (as you had predicted).

Let's enable it only for the github CI, similar to what we do for bazel:
https://github.com/tensorflow/tensorflow/blob/628fedebc4d6c28c881a353066d5493ed1c7a4ea/tensorflow/lite/micro/tools/ci_build/test_all.sh#L44-L58

With a comment that explains why we have made this choice:
```
// Enabling FVP for github CI only. This is because it currently adds ~4mins to each
// Kokoro run and is only relevant for external changes. Given all the other TFLM CI
// coverage, it is unlikely that an internal change would break only the corstone build.
```

Reply:
Done